redirects.js

```diff
@@ -0,0 +1,10 @@
+export default [
```

Comment:
Can we rename this to make it more obvious what kinds of redirects these are? Something like `renamed_path_redirects`?

Reply:
Sure, but I'm also concerned about the larger question of the fact that CI can't pass on this until the new pages exist in the repo, which requires publishing, and then re-publishing so the redirects go live.

Reply:
I think that's a good thing. It's catching bad redirects and won't publish.

I think the correct thing to do is to add the infra in this PR, and then update it in the docs sync PR when we sync renamed paths

Reply:
How would that work? Isn't that automatic?

Reply:
It is a good thing, but it means that there is an impedance mismatch between how docs publishing works and implementing redirects.

Reply:
I think having this file in the main repo and syncing it naively/wholesale would be hard to manage due to cherry-picks. For instance, we're adding a redirect here for 2.19 only:

- If we put the redirect file update into the `main` PR, then a release/doc sync from main won't work if that happens before the `2.19.x` changes are in
- If we put it into the 2.19 cherry pick PR, it will be overridden by the next main sync.

(I imagine we can make the automation more intelligent to solve this, but that means more complexity too.)

I wonder if we should have live chat about our approach to documentation and releases so we can get to shared direction more quickly rather than trying to hash it out over text.

Reply:
Yeah I was pondering this, and I think redirects are fundamentally not a versioned thing.